doc/userguide/rule-management/adding-your-own-rules.rst

```diff
@@ -58,3 +58,36 @@ Your alert should look like this:
 
   09/15/2011-16:50:27.725288  [**] [1:1:1] Do not read gossip during work [**]
   [Classification: Potential Corporate Privacy Violation] [Priority: 1] {TCP} 192.168.0.32:55604 -> 68.67.185.210:80
+
+Testing rules with Scapy
```

Comment:
alright, with your blessing I'd like to create a new page for testing/firing rules.